### ISSUE TITLE
UART Bridge Mode Issue: Fails to Open Serial Port in WebSerial

### ISSUE THREAD

**Issue by seuros** [hydrabus/hydrafw#171]
I am encountering an issue with the Hydrabus device when trying to use UART bridge mode. Specifically, I am unable to connect to the device using WebSerial via a browser. The connection attempt fails with the error "Failed to open Serial port."

**Steps to Reproduce**

- Connect the Hydrabus device to the computer.
- Switch the Hydrabus to UART bridge mode using the appropriate commands. 
  - uart 1
  - speed 115200
  - bridge
- Open a browser and navigate to a WebSerial application. example https://web.esphome.io
- Attempt to connect to the Hydrabus device.

**Expected Behavior**
I expect the browser to successfully connect to the Hydrabus device in UART bridge mode without any errors.

**Actual Behavior**
The browser fails to connect to the Hydrabus device, and I receive an error message stating "Failed to open Serial port."

Board: Hydrabus v1.0

**Comment by 0x8008135** [comment]
Hello,

Looks like you have a permission issue. I tested and could get it listed and even interact with the shell or uart bridge.

![Screenshot from 2024-05-01 14-02-59](https://github.com/hydrabus/hydrafw/assets/5058971/2b8f3884-6fcc-4f8e-ae8f-2c4109560d6a)

**Comment by bvernoux** [comment]
Yes I confirm I have tested on my side with Windows it works perfectly using Chrome browser (or any browser which support correctly WebSerial)

**Comment by seuros** [comment]
I confirm, Windows worked flawlessly.
 
For Linux i had to add the user to the dialout group, then reset the session.

```
$ sudo usermod -aG dialout $USER
```

Thanks
